app/config.py

```diff
@@ -412,3 +412,5 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
 
 PHONE_PROVIDER_2_HEADER = os.environ.get("PHONE_PROVIDER_2_HEADER")
 PHONE_PROVIDER_2_SECRET = os.environ.get("PHONE_PROVIDER_2_SECRET")
+
+ZENDESK_HOST=os.environ.get('ZENDESK_HOST', 'noone.zendesk.com')
```

Comment:
I wonder if we should use `os.environ.get('ZENDESK_HOST')` without providing a default value here? I think by default `ZENDESK_HOST` can be None which means the support page can't be used.

Reply:
Ok. I'll change so if the HOST is None then we show a page saying the support is disabled.